guide/private-key-management/backups.md

```diff
@@ -0,0 +1,256 @@
+
+Since adding a pass-phrase will generate a different wallet, it is best to add it when first creating the wallet, and before storing any bitcoin in its addresses.
+
+It's worth noting that most hardware wallets support pass-phrases, while most software wallet applications do not.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/backups/offline-discoveryprotection.png"
+   retina = "/assets/images/guide/private-key-management/backups/offline-discoveryprotection@2x.png"
+   alt-text = "Offline backup with discovery protection"
+%}
+
+
+**Do** 
+- Use a secure pass-phrase, it is case sensitive and can include numbers and special characters
+- Have redundant and secure backups of both the recovery-phrase and the pass-phrase, they are equally important for successful wallet recovery
+- Understand you now have two distinct wallets, (one each from the recovery-phrase with or without the pass-phrase), this can be useful to fool a bad actor, or a risk if next-of-kin is unaware of the pass-phrase 
+
+**Don’t** 
+- Store the recovery-phrase and pass-phrase together
+- Forget to document your setup for next-of-kin
+
+**Suitable for**
+- Large amounts
+- Situations where the backup can be discovered by bad actors
+
+**Alternative**
+
+Set up a hardware wallet from the recovery and pass-phrase that uses a PIN to protect usage. The PIN now becomes the protection against theft on discovery and should be saved somewhere safe just like the pass-phrase.
+
+<br>
+
+#### Level 4 
+## Multiple backups with discovery protection
+One remaining flaw with the single offline backup levels is that severe damage to the one location where they are stored, could lead to the backup not being usable. This could be a fire, flood, earthquake or similar, all more or less likely depending on where you live.
+
+To protect against such a *catastrophic event* we can create multiple backups, and distribute them to other locations. This could be a second home, trusted family and friends or a safe deposit box. 
+
+As the risk of discovery increases with the multiplying backups, it's best to maintain the protection against theft on discovery with a pass-phrase, or hardware wallets with a PIN. 
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/backups/multiple-discoveryprotection.png"
+   retina = "/assets/images/guide/private-key-management/backups/multiple-discoveryprotection@2x.png"
+   alt-text = "Multiple offline backups with discovery protection"
+%}
+
+**Do** 
+- Make sure you have reduntant copies of all parts of the backup in several locations
+- Document your setup well for next-of-kin now that complexity has increased
+
+**Don’t** 
+- Forget to safely store the pass-phrases or PINs
+
+**Suitable for**
+- Large amounts
+- In locations where catastrophic events are common, or likely to cause severe damage
+
+<br>
+
+# Inheritance instructions
```

Comment:
Should be an H2 header.

Reply:
You could use a horizontal ruler here, as you are starting a different content section of this page.